docs/Processes.md

```diff
@@ -555,6 +555,80 @@ The modifications from the standard tick.q include:
 
 The tickerplant log file will be written to hdb/database.
 
+<a name="segmentedtickerplant"></a>
+
+Segmented Tickerplant
+---------------------
```

Comment:
This section seems unnecessary. Could we just drop the segmented TP doco in here and lose this bit? Would need to make sure the segmented TP bit follows the other documentation for headers etc. (see the Gateway documentation in here, it goes to 3 levels)

Reply:
sorted